plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -88,6 +88,9 @@ public function __construct( ?int $p_process_urls = OFF, ?int $p_process_buglink
 		$this->setSafeMode( true );
 		# Only turn URLs into links if config says so
 		$this->setUrlsLinked( (bool) $this->config_process_urls );
+
+		$this->InlineTypes['@'] []= 'EmailText';
+		$this->inlineMarkerList .= '@';
```

Comment:
I was thrown off by the unusual space before `[]=`, was that intentional ? 
I would expect to see this written as `$this->InlineTypes['@'][] = ` instead.

Reply:
> I was thrown off by the unusual space before []=, was that intentional ?

Yes, I took this code from the Parsedown extension example and I liked that the framing with spaces `[]=` emphasises this as a "unary operation of adding a new element to the array", so I didn't remove the space. 😎 But I can change it if you want.